src/platform/android/DnssdImpl.h

```diff
@@ -32,8 +32,9 @@ void InitializeWithObjects(jobject resolverObject, jobject chipMdnsCallbackObjec
 /**
  * Pass results of the service resolution to the CHIP stack.
  */
-void HandleResolve(jstring instanceName, jstring serviceType, jstring address, jint port, jlong callbackHandle,
+void HandleResolve(jstring instanceName, jstring serviceType, jstring address, jint port, jobject attributes, jlong callbackHandle,
                    jlong contextHandle);
 
+const char * getTxtInfoKey(JNIEnv * env, jstring key);
```

Comment:
Add documentation on what this does. I am unclear reading the documentation and I am unclear if empty value should be handled (I see that returned as a fallback)

Reply:
Removed this from DnssdImpl.h and added a description in the cpp file.